server/src/main/java/org/opensearch/cluster/ClusterInfo.java

```diff
@@ -63,9 +65,10 @@ public class ClusterInfo implements ToXContentFragment, Writeable {
     public static final ClusterInfo EMPTY = new ClusterInfo();
     final Map<ShardRouting, String> routingToDataPath;
     final Map<NodeAndPath, ReservedSpace> reservedSpace;
+    final Map<String, FileCacheStats> nodeFileCacheStats;
```

Comment:
In theory, the cache size (across all nodes) could be pretty large but we don't need the whole `FileCacheStats`, we basically just need a single long `total` out of it, what if we introduce much smaller `FileCacheUsage` instead: 

```
public class FileCacheUsage {
    final long total;
}
```
?

Reply:
Sorry @reta! I missed this comment. That's fair. I was concerned of the unnecessary data transfer between nodes. 
Let me try to cook up something more optimized.